src/main/assembly/dist/md/ddm/ddm.xsd

```diff
@@ -207,7 +229,7 @@
 
     <xs:complexType name="LinkedRelationType">
         <xs:complexContent>
-            <xs:extension base="ddm:NonEmptyString"> 
+            <xs:extension base="ddm:NonEmptyString">
```

Comment:
these updates have not been done in the other file. Is that intentional? Shouldn't we keep them completely identical?